### ISSUE TITLE
Feature request - force ignored into Ignored Words file

### ISSUE THREAD

**Issue by robertson-bauercontrols** [EWSoftware/VSSpellChecker#274]
I just started using this product as the spell checking solution for our new product, there's one small thing I wish we could configure:

When you attempt to add a file to the ignored words dictionary, you have the option to put it globally.  It would be great if our solution layer could block that, so that all custom and ignored words go through the specified dictionaries.  Team members keep accidently adding ignored words to the global dictionary.

![image](https://user-images.githubusercontent.com/55808281/233392580-8bd5d43c-11bf-4dfc-b6fe-646cc0200e4d.png)

Thanks!

**Comment by EWSoftware** [comment]
There's an option to inherit ignored words in the configuration.  Currently, it only clears the ignored words and not the list of ignored words files.  I've fixed that for the next release.  If you turn off the inheritance option on the ignored words page, that will prevent it from showing the global ignored words file in the next release.  If you still want to inherit ignored words from the global file, you'd need to leave the inheritance option checked though.  However, I have changed the order of the files so that it lists them with those nearest the edited file first.  In effect this always puts the global file last on the submenu.
